app-admin/doctl/doctl-1.123.0.ebuild

```diff
@@ -1,13 +1,14 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-inherit bash-completion-r1 edo go-module
+inherit edo go-module shell-completion
 
 DESCRIPTION="A command line tool for DigitalOcean services"
 HOMEPAGE="https://github.com/digitalocean/doctl"
-SRC_URI="https://github.com/digitalocean/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/digitalocean/${PN}/releases/download/v${PV}/${P}-source.tar.gz"
```

Comment:
Could you please describe this change in commit message?

Additionally, could you please remove an extra empty line between `Closes:` and `Signed-off-by:` trailers in commit message to make the trailer block compact?

Reply:
I didn't expect that you will just prolong the commit message headline. We prefer up to 75 characters long lines in commit messages in gentoo if possible, see https://wiki.gentoo.org/wiki/Gentoo_git_workflow#Commit_message_format, and the headline here is unnecessarily longer. Could you please describe the change in the commit message body together with a motivation for the change? I believe that it is better to have this information in the commit message, because we don't have to later search through the b.g.o if we need to find out why the `SRC_URI` was change.

Reply:
ping